src/content/overview/developer-portal/_index.md

```diff
@@ -1,12 +1,33 @@
 ---
 title: Developer portal
-description: The central Giant Swarm self-service portal for your platform engineers and developers.
+description: Our Backstage-based developer portal is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so engineers using your platform find all the information they need in the right place.
 weight: 30
 menu:
   principal:
     parent: overview
     identifier: overview-developer-portal
-last_review_date: 2024-03-18
+last_review_date: 2024-07-01
 owner:
-  - https://github.com/orgs/giantswarm/teams/sig-product
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
 ---
+
+Our developer portal absed on [Backstage](https://www.cncf.io/projects/backstage/) is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.
+
+Giant Swarm can assist you in various ways depending on where you are in your journey towards your own developer portal. We provide a pre-configured Backstage instance as a starting point to customers who don't run their own instance yet. We also provide plugins that can be used to extend the functionality of your Backstage instance.
+
+## Why developer portals
```

Comment:
Can we reword this? It sounds like an incomplete sentence where a verb should follow.

P.S. "Papa, kann ich Schokolade?" – "'...wegschmeißen' oder was?"

Reply:
What about leaving the content under the intro removing this header?